guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>The tool output above shows that some data points have been located outside New Orleans because of missing or invalid geometries. We want to explore data points within New Orleans city limits. As such, we want to run the tool only in the zoomed extent. Let's set our area of interest as the zoomed extent of map. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Let's set our area of interest as the zoomed extent of the map.</p>